draft-mavrogiannopoulos-tls-cid.md

```diff
@@ -207,6 +208,12 @@ Ideally, we would just bump the version number, but there seems to be limited ro
 
 More discussion needed to sort out this point.
 
+## De-duplication Algorithm
+
+The following algorithm assumes that receivers have an unambiguous way to tell that the wire format is the one described in {{sec:new-record-fmt}}.  As suggested in {{sec:new-wire-fmt}}, this could be signalled by a different version number { TBD, TBD }.
+
```

Comment:
{ 3, 4 } is not going to be used for 1.3, therefore it'd seem to me the most plausible choice?